dart/lib/src/sentry_options.dart

```diff
@@ -16,6 +16,9 @@ import 'platform_checker.dart';
 const defaultEnvironment = 'production';
 
 /// Sentry SDK options
+// TODO: Scope observers, enableScopeSync
+// TODO: shutdownTimeout, flushTimeoutMillis
+// https://api.dart.dev/stable/2.10.2/dart-io/HttpClient/close.html doesn't have a timeout param, we'd need to implement manually
```

Comment:
this reads weird, should not be part of the class docs (its an internal comment), I'd move this above `/// Sentry SDK options` and have a blank line in between